app/Http/Controllers/UserController.php

```diff
@@ -54,9 +54,15 @@ public function store(UserCreateRequest $request)
                 $message->to($email)->subject('Welcome to Chatwork Forwarder Application');
             });
             return redirect()->route('users.edit', $user)
-                             ->with('messageSuccess', 'This user successfully created');
+                ->with('messageSuccess', [
+                    'status' => 'Create success',
+                    'message' => 'This user successfully created'
+                ]);
         } catch (QueryException $exception) {
-            return redirect()->back()->with('messageFail', 'Create failed. Something went wrong')->withInput();
+            return redirect()->back()->with('messageFail', [
+                'status' => 'Create success',
+                'message' => 'Create failed. Something went wrong'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 64</br>Each line in an array declaration must end in a comma